cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/Schedulers.java

```diff
@@ -56,7 +56,8 @@
  * Common utility methods for scheduling.
  */
 public class Schedulers {
-  public static final String STORE_TYPE_NAME = ProgramScheduleStoreDataset.class.getName();
+  // we use the simple name for the type, due to CDAP-11632
```

Comment:
Don't comment with JIRAs that will be fixed with the current PR. you can add external JIRAS or Todo JIRAS.